concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/15 Customize Adaptive Detail Row.md

```diff
@@ -0,0 +1,46 @@
+Adaptive detail row contains the [Form UI component](/Documentation/ApiReference/UI_Components/dxForm/). You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing') handler to customize it. The following code marks the form item whose data field is *"OrderID"* as required:
```

Comment:
- "Adaptive detail row contains" - you haven't introduce the adaptive detail row.
- "customize it" - customize what? the row? the Form? the handler?
- What should I change in the handler to customize the Form?